src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  text = 'Nothing was pressed yet';
```

Comment:
```suggestion
  initialText = 'Nothing was pressed yet';
```
Let's clarify which text we are talking about